bindings/nodejs/package.json

```diff
@@ -59,15 +59,12 @@
     "node": ">= 10"
   },
   "scripts": {
-    "artifacts": "napi artifacts",
-    "build": "napi build --platform --release --js generated.js && node ./scripts/header.js",
-    "build:debug": "napi build --platform --js generated.js && node ./scripts/header.js",
+    "build": "napi build --platform --target \"${NAPI_TARGET:-}\" --release --js generated.js && node ./scripts/header.js",
+    "build:debug": "napi build --platform --target \"${NAPI_TARGET:-}\" --js generated.js && node ./scripts/header.js",
     "docs": "typedoc",
     "format": "prettier --write .",
-    "prepublishOnly": "napi prepublish -t npm",
```

Comment:
We should not delete this. This is useful for publishing.

Reply:
We will never do publish by hand, so their no need for such script in `package.json`.

Reply:
No. It is part of the publish lifecycle. 
https://docs.npmjs.com/cli/v9/using-npm/scripts#prepare-and-prepublish

Reply:
Oh, nice catch. I will add it back. (magic npm...)